### PR TITLE
Remove quickstart from marketing landing

### PR DESCRIPTION
**Pull request by ddc22**
#### Changes proposed in this Pull Request

* Remove quickstart from marketing landing

#### Testing instructions
* Go to `/marketing/do-it-for-me/landing`
* Make sure the text is updated

image

### CODE REVIEW

File: client/my-sites/marketing/do-it-for-me/landing.tsx [hunk 1]
```diff
@@ -93,7 +93,7 @@ export default function DoItForMeLandingPage(): JSX.Element {
 
 			<Paragraph>
 				{ translate(
-					'Need a professionally designed website for your business fast? Choose from our custom-designed templates below, add your content and media, and in just 2-4 business days your website will be ready to launch for just %(cost)s. The one-time fee includes a 4-page website and a 30-minute Quick Start Zoom orientation about your site when it’s complete.',
+					'Need a professionally designed website for your business fast? Choose from our custom-designed templates below, add your content and media, and in just 2-4 business days your website will be ready to launch for just %(cost)s. The one-time fee includes a 4-page website.',
```
Comment (a8ci18n) [inline client/my-sites/marketing/do-it-for-me/landing.tsx:96]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **16** times:
`translate( 'Need a professionally designed website for your business fast? Choose from our custom-designed templates below, add your content and media, and in just 2-4 business days your website will be ready to launch for just %(cost)s. The one-time fee includes a 4-page website and a 30-minute Quick Start Zoom orientation about your site when it’s complete.' )` *ES Score: 7*

**Review by dzver — APPROVED** [review]
LGTM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/6951414

Thank you @jdc91 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/6951414).

**Pull request merged**